backend/routes/login.js

```diff
@@ -7,18 +7,27 @@ const router = express.Router();
 router.post("/login", async (req, res) => {
   const { email, password } = req.body;
   // res.send({ message: "received in the backend " });
-
+console.log(res)
   // find that user in the database
   const foundUser = await Principle.findOne({ email, password }).exec();
-
+   
+   
   if (foundUser) {
 
     // send the user information to the front end including names , family info .. etc 
     console.log("foundit");
+    console.log(foundUser)
+    res.send(foundUser)
```

Comment:
It is recommended to send the status code from the backend. You can check here https://developer.mozilla.org/en-US/docs/Web/HTTP/Status